assets/src/app/AppLayout/Navigation.tsx

```diff
@@ -67,11 +67,8 @@ interface NavigationState {
 
 const staticNavigation:(MyRouterItem)[] = [
     {label: 'Getting Started', to: '/ato/getting_started'},
-    {label: 'Documents', routes: [
-        {label: 'Overview', to: '/ato/documents'},
-        {label: 'Vulnerability Management', to: '/ato/documents/vulnerability-management-plan'},
-        {label: 'Security Awareness', to: '/ato/documents/security-awareness-and-training-plan'},
-        {label: 'FedRAMP Templates', to: '/ato/documents/fedramp-templates'},
+    {label: 'FedRAMP Resources', routes: [
+        {label: 'SSP Templates', to: '/ato/documents/fedramp-templates'},
```

Comment:
Removing the old ``Documents`` section was intentional. Docs haven't been maintained in year(s) and were never completed to begin with.